udata_hydra/utils/http.py

```diff
@@ -0,0 +1,19 @@
+import aiohttp
+import logging
+
+from udata_hydra import config
+
+log = logging.getLogger("udata-hydra")
+
+
+async def send(dataset_id: str, resource_id: str, document: dict) -> None:
+    if not config.UDATA_URI or not config.UDATA_URI_API_KEY:
+        log.error(g"Missing Udata URI and API key to send http query")
+        return
+    uri = f"{config.UDATA_URI}{dataset_id}/resources/{resource_id}/extras/"
+    headers = {"content-type": "application/json", "X-API-KEY": config.UDATA_URI_API_KEY}
+    async with aiohttp.ClientSession() as session:
+        async with session.put(uri, json=document, headers=headers) as resp:
```

Comment:
`document` could have some `None` values, ex:
```
{
    'analysis:error': None,
}
```
However, udata extras can't be None.
We should either :
* allow None values in udata extras
* prevent sending None values here
* filter None values out on extras endpoint

Reply:
Done in https://github.com/etalab/udata-hydra/pull/27/commits/5ecaba7dbba7ad3cd7661a2e8a352aa8ef1d848b